cmd/ig/utils/flags.go

```diff
@@ -57,13 +57,22 @@ type CommonFlags struct {
 	// ContainerdNamespace is the namespace used by containerd
 	ContainerdNamespace string
 
-	// UseCri specifies whether to use the CRI API to talk to the runtime
-	UseCri bool
+	// RuntimeProtocol specifies whether to use the CRI API to talk to the runtime
+	// Supported values: internal, cri.
```

Comment:
Should we specify here that it makes sense only for docker and containerd? For CRI-O and Podman there is no choice. CRI-O will only use CRI and Podman always its internal protocol.